spotinst/commons/common_set_credential_aws.go

```diff
@@ -0,0 +1,88 @@
+package commons
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/spotinst/spotinst-sdk-go/service/account/providers/aws"
+	"log"
+)
+
+const (
+	SetCredentialAWSResourceName ResourceName = "spotinst_set_credential_aws"
```

Comment:
change resource name to spotinst_credentials_aws

Reply:
updated name to spotinst_credentials_aws